src/status_im/utils/config.cljs

```diff
@@ -20,4 +21,8 @@
 (def notifications-wip-enabled? (enabled? (get-config :NOTIFICATIONS_WIP_ENABLED 0)))
 (def stub-status-go? (enabled? (get-config :STUB_STATUS_GO 0)))
 (def mainnet-networks-enabled? (enabled? (get-config :MAINNET_NETWORKS_ENABLED 0)))
+(def log-level
```

Comment:
One idea is also to make this like enums, something like in go-ethereum / other big OSS:

 --verbosity value Logging verbosity: 0=silent, 1=error, 2=warn, 3=info, 4=core, 5=debug, 6=detail (default: 3)
https://github.com/ethereum/go-ethereum/wiki/Command-Line-Options

Reply:
I'm not sure what exactly this will improve

Reply:
Mostly for external contributors I guess. Make it clear what the scale is when you are trying things out, as opposed to having to be aware of log level only being debug or error, and if you type info you get error by default. Similar to adding extra `v` `-vvv` when debugging SSH.

Reply:
I mean, i understand why this can be useful in cli tools or something similar, in our case this literally 1) changes nothing 2) i will need to make an extra effort in order to map the number with that keyword... so still not sure why we need this.

Anyway that can be a subject for another PR.

Reply:
I mean an extra effort when reading configuration :)

Reply:
Agree (hence approval); my comments were more about improving logging in general for the future.

Reply:
As for why: I think biggest win in medium-term future would be syncing with status-go logs.

Reply:
> I mean an extra effort when reading configuration :)

This is true. Unlike enums they aren't immediately swappable for numbers without a bunch of glue code